pages/index.tsx

```diff
@@ -1,17 +1,38 @@
-import React from 'react'
-import Features from '../components/index/Features'
-import Header from '../components/index/Header'
-import AddToSlackForm from '../components/index/AddToSlackForm'
-import Footer from '../components/Footer'
-
+import React from "react";
+import Features from "../components/index/Features";
+import Header from "../components/index/Header";
+import AddToSlackForm from "../components/index/AddToSlackForm";
+import Footer from "../components/Footer";
+import FeatureImageBox from "../components/FeatureImageBox";
+import { FPLBOT_APP_URL } from "../utils/envconfig";
 
 export default function Index() {
   return (
     <div>
       <Header />
       <Features />
       <AddToSlackForm />
+      <a
+        href={`${FPLBOT_APP_URL}/search`}
```

Comment:
Could simply be `/search` perhaps (just like `/search.png` on line 28). Clicking the link in the PR environment took me to a 404 in prod, so making it relative instead would solve that, I guess.

Reply:
🤦‍♀️ Jeg er dum

Reply:
Så vi kan egentlig fjerne `FPLBOT_APP_URL` helt, og bare bruke relative urls..?

Reply:
burde gå bra det ja, bare hjernen min som ikke fungerte når jeg implementerte det 😄